packages/manager/modules/pci/src/projects/project/storages/databases/add/add.controller.js

```diff
@@ -224,7 +184,7 @@ export default class {
   }
 
   onAdditionalDiskChange() {
-    this.$timeout(() => this.getOrderData());
+    this.$timeout(() => this.getOrderDataFromModel(this.model));
   }
```

Comment:
This function is useless: 
- `getOrderData` was used to update a few vars, but is not needed anymore. 
- `getOrderDataFromModel`  is only a converting function between the page's model and the API expected format, it does not make sense to do the conversion on the change of additional disk.
You can delete it from the controller, and remove the binding in the template file too.

Reply:
Fixed :)